fhirAssembly/src/main/resources/assembly/com.vantiq.fhir.fhirConnection/procedures/com/vantiq/fhir/fhirService_returnLink.vail

```diff
@@ -0,0 +1,39 @@
+package com.vantiq.fhir
+
+import service com.vantiq.fhir.fhirService
+
+/**
```

Comment:
Since this is a "top level-esque" comment, it should probably be moved to be before the "package" statement.  That way it will be treated as the procedure's description instead of just a random embedded comment (we can discuss whether that placement should be necessary, but currently it is, so...).